docs/Guide/plugins/Subcommands/getting-started.mdx

```diff
@@ -16,29 +16,54 @@ npm install @sapphire/plugin-subcommands @sapphire/framework @sapphire/utilities
 
 ### Usage
 
-```typescript ts2esm2cjs|{7-8}|{7-8}
-import { SubCommandPluginCommand } from '@sapphire/plugin-subcommands';
+```typescript ts2esm2cjs
+import {
+  SubCommandPluginCommand,
+  MessageSubcommandMappings,
+  ChatInputSubcommandMappings,
+  SubcommandMappingsArray
+} from '@sapphire/plugin-subcommands';
 import type { Args } from '@sapphire/framework';
-import type { Message } from 'discord.js';
+import type { Message, CommandInteraction } from 'discord.js';
 
 // Extend `SubCommandPluginCommand` instead of `Command`
 export class UserCommand extends SubCommandPluginCommand {
-  public constructor(context: SubCommandPluginCommand.Context, options: SubCommandPluginCommand.Options) {
```

Comment:
curious on the removal of `public` here as it's used throughout the rest of the file

Reply:
added it back